### DISCUSSION TITLE
Metrics collection

### DISCUSSION THREAD

**Discussion by dr460neye** [kube-green/kube-green#371] [Ideas]
A nice idea would be to centrally log the "saved CO2" by sending back the amount of Co2 eq saved in total as anonymous user metrics.
What do you think?

**Comment by davidebianchi** [comment]
Hi @dr460neye! Interesting idea, but it requires to have something running the collector of this data, but at the moment there is nothing centrally running.
Or do you have any other ideas in mind?
